py/server/tests/test_pt_proxy.py

```diff
@@ -209,14 +209,14 @@ def test_as_of_join(self):
                 joined_pt_proxy = pt_proxy.aj(right_table, on=["a"])
                 self.assertTrue([ct for ct in joined_pt_proxy.target.constituent_tables if ct.size > 0])
 
-                joined_pt_proxy = pt_proxy.aj(right_table, on=["a"], joins="e", match_rule=AsOfMatchRule.LESS_THAN)
+                joined_pt_proxy = pt_proxy.aj(right_table, on=["a < a"], joins="e")
```

Comment:
@jjbrosnan @margaretkennedy This PR is a breaking change.  If the last column is named the same, you get expressions like these, which users are going to find weird.  We need to carefully address this in our docs.